.travis.yml

```diff
@@ -1,14 +0,0 @@
-language: android
-jdk: oraclejdk8
-sudo: true
-android:
-  components:
-    - tools
-    - platform-tools
-    - build-tools-30.0.2
-    - android-30
-    - extra-google-m2repository
-before_install:
-  - yes | sdkmanager "platforms;android-30"
-script: ./gradlew --stacktrace clean lintRelease testRelease
```

Comment:
I noticed that we're not doing `lintRelease` in the GitHub Actions workflow -- just curious why that is.

Reply:
Yeah we never actually do anything with the output. The best lint tool so far has been Android studio. Ideally we would want the lint output in a dashboard and we can track lint over time.